parts/language/dict_en.lua

```diff
@@ -570,567 +570,568 @@ FNNS and     {"Support 3",
     },
     {"Techmino attack",
         "techminoattack",
         "term",
-        "Check the user manual on the bottom right corner of the homepage for more information.",
+        "See the User Guide in the lower right corner of the home page.",
     },
     {"C2 Generator",
         "cultris2generator cultrisiigenerator c2generator",
         "term",
-        "All Tetrominoes have an initial weight of 0.\nEvery time, divide all weights by 2, add a random number between 0 and 1, pick the piece with the highest weight, and divide this piece’s weight by 3.5.",
+        "The generator in Culturis 2 uses the following algorithm to generate block sequences:\n1. Set the weight of all tetrominoes to 0 at the beginning of the game.\n2. Divide all weights by 2, then add a random number (0-1) to all weights.\n3. Pick the tetromino with the highest weight, then divide that tetromino's weight by 3.5.\n4. Repeat from step 2.",
+        -- Original Lua file contains this comment: "Discovered by zxc"
     },
     {"Stacking",
         "stacking",
         "term",
-        "Often refers to stacking Tetrominoes without leaving holes. An essential skill.",
+        "Stacking the tetrominoes without leaving any holes. An essential skill that requires a good next-reading ability. It can be practiced by playing 40L with only Tetrises and no Hold.",
     },
     {"Rotation buttons (1/2)",
         "doublerotation",
         "term",
-        "Using both clockwise and counter-clockwise rotation buttons reduces the number of key presses by replacing three rotation presses with one press of the opposite direction.\nFinesse assumes the use of both rotation buttons.",
+        "Using both rotation buttons (clockwise and counterclockwise) would reduce the number of keystrokes, since rotating three times in one direction is the same as rotating once in the opposite direction. This is also a prerequisite for learning finesse.",
     },
     {"Rotation buttons (2/2)",
         "triplerotation",
         "term",
-        "Using all three rotation buttons (the third being 180° rotation), any piece only requires one rotation press to reach the desired direction.\nHowever, it is not exactly useful for not every game has this feature, and the speed increase from learning this technique is not as much as when you learn using both rotation buttons as opposed to one. You can skip this technique unless you want extreme speeds.",
+        "By using all three rotation buttons (adding 180° rotation), one can use one keypress to rotate any block into its desired orientation (when spins are not considered). \nHowever, we do not recommend that you spend too much time practicing with all three rotation buttons. This is because not all Tetris games have 180° rotations, and using three rotations would not increase your speed significantly compared to using just two. It is not a necessary skill unless you want to pursue extremely high speed.",
```

Comment:
Hmm can we merge 2 items of ``Rotation buttons`` into one? (We may already have #935 (before is #926) right?)